infra/environments/test/terraform.tfvars

```diff
@@ -1,5 +1,11 @@
 role_arn = "arn:aws:iam::077346344852:role/terraform_deploy_role"
 
+# lambda
 image_name    = "hello_world"
 data_bucket   = "test-data"
 function_name = "hello-lambda2"
+
+# downloads/dns
+hosted_zone            = "curibio-test.com"
+ssl_verification_name  = "_604b45423128aa2b456bfc5d0368be2b.curibio-test.com."
```

Comment:
is the reason that `prod` and `modl` don't have these is that they won't be known until after we merge and create the cloudfront distribution? then we'll need to do a new PR that includes those SSL certs and merge it through?  if so, that seems like an acceptable one-time cost for setting up a new SSL certificate (as opposed to somehow trying to completely automate it), but just wanted to verify if my understanding is correct

Reply:
actually I can remove that now since I moved the cert creation to terraform so the only part that isn't managed is creating the domain registration